jsk_pr2_robot/jsk_pr2_startup/pr2_bringup.launch

```diff
@@ -5,6 +5,7 @@
   <arg name="launch_wifi_ddwrt" default="false" />
   <arg name="launch_network_detector" default="false" />
   <arg name="launch_app_manager" default="true" />
+  <arg name="launch_mannequin_mode" default="true" />
```

Comment:
Just a comment.
This launch file __loads__ arm / head controllers for mannequin mode but does __not start__ them, right?
If so, `launch_mannequin_mode` is a little confusing because someone may think it will __load and start (spawn)__ controllers for mannequin mode.